lib/views.js

```diff
@@ -438,7 +437,7 @@ exports.tasks_messages = {
             // used for fetching a specific message based on uuid
             emit(msg.uuid, val);
             // used for querying latest tasks in a specific state
-            emit([task.state, when], val);
+            emit(task.state, val);
```

Comment:
I think it's useful to keep the `when` value - it sorts the results so we can send the oldest first.